### ISSUE TITLE
Make it clear that it doesn't need <script setup> to work

### ISSUE THREAD

**Issue by gustojs** [antfu/unplugin-auto-import#41]
I've noticed some misunderstandings in the community that `unplugin-auto-import` requires using `<script setup>` syntax, based on the fact that the code examples are using it exclusively.
It could possibly be helpful to state that it's not necessary and it works just fine without it.

**Comment by antfu** [comment]
Yes, indeed. I just notice we didn't have many statements to explain what this plugin is trying to do (we actually work in any js/ts files). I am open to PR if anyone what to kickstart it.

**Comment by antfu** [comment]
I don't think we want it atm. Vitesse could already be a good index / showcases of them
